apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -222,9 +223,9 @@ const DisplayServiceAccountCard = (props: Props) => {
       ga4PropertiesError;
     if (configError) {
       return isFirstSetup ? (
-        <Badge variant="warning">Incomplete</Badge>
+        <Badge variant="primary">Finish configuration steps below</Badge>
       ) : (
-        <Badge variant="negative">"Some checks were unsuccessful"</Badge>
+        <Badge variant="negative">Problems with your configuration</Badge>
```

Comment:
let's remove `your` here and try to stop using that pronoun. WDYT about just `Problem with configuration` or even `Problem with configuration detected`?

Reply:
This cleans up the 3 possibilties to:

**Status:**

* <Badge variant="primary">Finish configuration steps below</Badge>
* <Badge variant="negative">Problems with configuration</Badge>
* <Badge variant="positive">Successfully configured</Badge>

Reply:
👍 sgtm